### PR TITLE
Add development flow example; Add url link for dev admin portal;

### PR DESCRIPTION
**Pull request by jotshin**
I made the following things in this PR:

- Move `Things to Remember in First Day.md` to higher order so that any future new team member could have quicker access to it
- Include a development flow example for knowing briefly how a common iOS development flow could be

Here is [the reference Dropbox link](https://paper.dropbox.com/doc/Conichi-Onboard-Feedback-RwpOShCdytd6jDiHRDkRa)

@David-Henner @ShengHuaWu please help me to correct if there's anything wrong or to improve. Thanks.

### CODE REVIEW

File: docs/things_to_remember.md [hunk 1]
```diff
@@ -11,9 +11,42 @@ We use [Confluence](https://www.atlassian.com/software/confluence) for documenta
...
 
-- Clone This repository ([ios-team](https://github.com/conichiGMBH/ios-team)) which include all guidelines of our team and installation scripts for Homebrew, openssl, Xcode command line tools, gnupg, rvm, ruby, bundler etc. Scripts also clone all necessary repos and bootstrap them. 
+- Clone This repository ([ios-team](https://github.com/conichiGMBH/ios-team)) which include all guidelines of our team and installation scripts for Homebrew, openssl, Xcode command line tools, gnupg, rvm, ruby, bundler etc. Scripts also clone all necessary repos and bootstrap them.
 
 - Don't hesitate and feel free to ask any question!
 
+#### Development Flow Example
+
+- Confluence reference for business logic and user story for the given feature:    
+  - [Main UI reference](https://conichi.atlassian.net/wiki/spaces/CON/pages/52765598/Documentation+Core+Resources#Documentation(CoreResources)
```
Comment (ShengHuaWu) [inline docs/things_to_remember.md]: The syntax of this line needs modifying.

File: docs/things_to_remember.md [hunk 2]
```diff
@@ -11,9 +11,42 @@ We use [Confluence](https://www.atlassian.com/software/confluence) for documenta
...
+
+- [Git protocol](https://github.com/conichiGMBH/ios-team/blob/master/docs/git_protocol.md) management:
+    - For features → You can create `Sub-Task` from the `Story`, and if the sub-task PR is approved, rebase it back the the story and then push to Github
+    - On Github press delete the branch after PR approved and rebase is done
+
+
+- Pay attention to repo `README.md` and `RELEASING.md`
+- Use `VIPER` architecture to code
+- You could also ask for Sketch file for UI elements
+- [Lokalise](https://www.google.com.tw/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0ahUKEwjorq-c3anYAhUIUZQKHVycCa8QFgglMAA&url=https%3A%2F%2Flokalise.co%2F&usg=AOvVaw1aqyIn0HM4aBwZzfhfL7Hk) for localized strings
```
Comment (ShengHuaWu) [inline docs/things_to_remember.md]: It might be better to use `https://lokalise.co/` for simplicity.

File: docs/things_to_remember.md [hunk 3]
```diff
@@ -11,9 +11,42 @@ We use [Confluence](https://www.atlassian.com/software/confluence) for documenta
...
+- [API reference](https://github.com/conichiGMBH/api-documentation)
```
Comment (ShengHuaWu) [inline docs/things_to_remember.md]: Given the message from our backend team, this repository will be closed in the future.
Therefore, we have to go back and change the link after changing.

**Comment by jotshin** [comment]
@ShengHuaWu It's been updated according to your comments. Please let me know if you have other questions. Thanks.

File: docs/things_to_remember.md (hunk 1 above)
Comment (jotshin) [reply docs/things_to_remember.md]: Updated in 252ed2d3c8f87456cb786083a9eea862f47cfff9

File: docs/things_to_remember.md (hunk 2 above)
Comment (jotshin) [reply docs/things_to_remember.md]: Updated in 252ed2d3c8f87456cb786083a9eea862f47cfff9

File: docs/things_to_remember.md (hunk 3 above)
Comment (jotshin) [reply docs/things_to_remember.md]: Updated in 252ed2d3c8f87456cb786083a9eea862f47cfff9

File: docs/things_to_remember.md [hunk 4]
```diff
@@ -11,9 +11,43 @@ We use [Confluence](https://www.atlassian.com/software/confluence) for documenta
...
+[1]: https://conichi.atlassian.net/wiki/spaces/CON/pages/52765598/Documentation+Core+Resources#Documentation(CoreResources)
+
+- Jira ticket for task and schedule management:
+    - For features → PM will assign to you or you could assign yourself to `in progress`
+    - For issues → assigned by QA or PM
+    - There’s a type field for indicating `Story` and `Sub-Task`. You could also change a `Bug` to `Story` for including different needed `Sub-Task`
+
+
+- [Git protocol](https://github.com/conichiGMBH/ios-team/blob/master/docs/git_protocol.md) management:
+    - For features → You can create `Sub-Task` from the `Story`, and if the sub-task PR is approved, rebase it back the the story and then push to Github
```
Comment (David-Henner) [inline docs/things_to_remember.md]: maybe `rebase it back in the story` ?

File: docs/things_to_remember.md [hunk 5]
```diff
@@ -11,9 +11,43 @@ We use [Confluence](https://www.atlassian.com/software/confluence) for documenta
...
+
+
+- Pay attention to repo `README.md` and `RELEASING.md`
+- Use `VIPER` architecture to code
+- You could also ask for Sketch file for UI elements
+- [Lokalise](https://lokalise.co/) for localized strings
+- [API reference](https://github.com/conichiGMBH/api-documentation) (this could be changed in the future)
+- Write test:
+    - Quick / Nimble / Nimble Snapshot for Swift
+    - OCMock / FBSnapshotTests for Objective-C
```
Comment (David-Henner) [inline docs/things_to_remember.md]: If I'm not mistaken, we also use Quick and Nimble in Objective-C

File: docs/things_to_remember.md (hunk 4 above)
Comment (jotshin) [reply docs/things_to_remember.md]: Thanks for your correction. 😆 
Corrected in https://github.com/conichiGMBH/ios-team/pull/58/commits/d83da8a94e8b4145ac4df116a204ee4e8d812651

File: docs/things_to_remember.md (hunk 5 above)
Comment (jotshin) [reply docs/things_to_remember.md]: You're absolutely right.
Updated in https://github.com/conichiGMBH/ios-team/pull/58/commits/d83da8a94e8b4145ac4df116a204ee4e8d812651

**Pull request merged**